deployments/properties/k8s.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to a Kubernetes Cluster",
```

Comment:
should we have this "Deploy to a AKS cluster using KubeConfig"

Reply:
no -- this is intentionally generic, as it works for any K8s cluster.  Adding "using kubeconfig" seems redundant, as that is always required to interact with the k8s api server